bot/exts/moderation/verification.py

```diff
@@ -109,6 +109,25 @@ def is_verified(member: discord.Member) -> bool:
     return len(set(member.roles) - unverified_roles) > 0
 
 
+async def safe_dm(coro: t.Coroutine) -> None:
```

Comment:
This seems like it'll be useful elsewhere. Therefore, I think its home should be in utils/messages or whatever it's named. However, the move could be saved for when someone goes through all other cases to which this function could be applied.

Reply:
I feel like this would be better as a context manager, but a function is okay.

Reply:
You're right, it would make sense for it to be a context manager. Although it adds an extra level of indentation if you're still catching other errors coming out of it, but that's a weak excuse.

I'd say let's leave it as a func for now and start thinking of how we can design a more generic suppressor that can eventually replace this and be used elsewhere as well. It may be useful to have the error code configurable, e.g.:

```python
with suppress_error_code(Codes.DISABLED_DM):
  ...
```

I'm not sure whether we'd need the error code configurable though, it may just be the case that `50_007` is the only one we realistically need to handle in a specific way.